src/cmd/alias.go

```diff
@@ -1,8 +1,8 @@
 package cmd
 
 import (
-	"github.com/Originate/git-town/lib/git"
-	"github.com/Originate/git-town/lib/script"
+	"github.com/Originate/git-town/src/git"
```

Comment:
With all the other folders we have at the top level, I just thought it would be nice to have all the go code in a single folder instead of two. I also haven't been using the generator and just copying other commands.